### PR TITLE
Update ome-faq.md

### PR DESCRIPTION
**Pull request by samschan-ms**
@KCCross  please review and merge

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ome-faq.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ If a file format is supported, such as a Word, Excel, or PowerPoint file, the fi
   
 ## Are PDF file attachments supported?
 
-The short answer is yes! PDF encryption allows you to protect sensitive PDF documents through secure communication or secure collaboration. When you send email, the Office 365 service encrypts PDF file attachments not the Outlook client.
+The short answer is yes! If enabled, PDF encryption allows you to protect sensitive PDF documents through secure communication or secure collaboration. When you send email, the Office 365 service encrypts PDF file attachments not the Outlook client.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
The short answer is yes! If enabled, PDF encryption allows you to protect sensitive PDF documents through secure communication or secure collaboration. When you send an email, the Office 365 service encrypts PDF file attachments, not the Outlook client.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a couple of additional edits, please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**